### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by yalh76**

### CODE REVIEW

**Comment by yalh76** [comment]
!testme

**Comment by yalh76** [comment]
!testme

**Comment by yalh76** [comment]
!testme

**Comment by yalh76** [comment]
!testme

**Comment by yalh76** [comment]
!testme

**Comment by yalh76** [comment]
!testme

**Comment by yalh76** [comment]
!testme

**Comment by yalh76** [comment]
!testme

**Comment by yalh76** [comment]
!testme

**Comment by yalh76** [comment]
!testme

**Comment by yalh76** [comment]
!testme

File: scripts/restore [hunk 1]
```diff
@@ -15,157 +17,139 @@ source /usr/share/yunohost/helpers
...
+ynh_script_progression --message="Validating restoration parameters..."
 
-domain=$(ynh_app_setting_get $app domain)
-#path_url=$(ynh_app_setting_get $app path) #not used
-#admin=$(ynh_app_setting_get $app admin) #not used
-is_public=$(ynh_app_setting_get $app is_public)
-#language=$(ynh_app_setting_get $app language) #not used
-nonfree=$(ynh_app_setting_get $app nonfree)
-
+ynh_secure_remove --file="$final_path"
```
Comment (ericgaspar) [inline scripts/restore:38]: Are we sure to remove the final_path here? 🤔

**Review by ericgaspar — CHANGES_REQUESTED** [review]

File: scripts/restore (hunk 1 above)
Comment (yalh76) [reply scripts/restore:38]: I don't know, but it was done like that => L36 `ynh_secure_remove $final_path`

**Comment by yalh76** [comment]
!testme

**Comment by yalh76** [comment]
was too difficult to make it works, I prefer to restart from scratch on https://github.com/YunoHost-Apps/zabbix_ynh/tree/example

**Pull request closed without merging**